bodhi/tests/client/test___init__.py

```diff
@@ -501,17 +508,27 @@ def test_url_flag(self, send_request):
                 return_value=client_test_data.EXAMPLE_UPDATE_MUNCH, autospec=True)
     @mock.patch('__builtin__.raw_input', create=True)
     def test_queryoverrides_mine_flag_username_unset(self, mock_raw_input, send_request):
-        """
-        Assert that we use init_username if USERNAME is not set
-        """
+        """Assert that we use init_username if --user is not given."""
         mock_raw_input.return_value = 'dudemcpants'
-        runner = testing.CliRunner()
+        real_open = open
 
-        runner.invoke(client.query_buildroot_overrides, ['--mine'])
+        def fake_open(*args, **kwargs):
+            """Fake open so that it looks like we have no session cache."""
+            if args[0] == fedora.client.openidbaseclient.b_SESSION_FILE:
+                return mock.mock_open(read_data='{}')(*args, **kwargs)
+            return real_open(*args, **kwargs)
```

Comment:
Do you want to see about re-using the function defined above?

Reply:
@pypingou Sure, I'm pushing a different commit that shares the function. Thanks for the suggestion!